src/sampler.jl

```diff
@@ -55,7 +55,7 @@ end
 
 function boxregion_multgauss(as, bs, rng)
     @assert length(as) == length(bs) > 0
-    center = mean(hcat(as,bs), dims=2)
+    center = mean(hcat(as,bs))
```

Comment:
The reason this was not worrking is because (for performance reasons) we make the data a `StateSpaceSet` which is a vector of `SVectors`. You can define mean across the vectors as you can sum them. `dims = 2` makes no sense in this context, it was left over from backw hen things were a matrix with each column a vector.